test/step-definitions/requests.ts

```diff
@@ -52,6 +53,27 @@ export class requests {
     await this.context.app.init();
   }
 
+  @given(/authorization with Reading-Scopes/)
+  public async generateReadToken() {
+    const token = sign(
+      { scopes: ['Data.Read'], serverId: 'eddiehub' },
+      process.env.SECRET,
+    );
+    this.context.bearerToken = token;
+  }
+
+  @given(/authorization with Writing-Scopes/)
+  public async generateWriteToken() {
```

Comment:
What do you think about merging these 2 functions into one?

From:
- `authorization with Reading-Scopes`
- `authorization with Writing-Scopes`
To:
- `authorization with "Reading-Scopes|Writing-Scopes"`

Reply:
yes I planned to do that, but didnt find time yet to implement it :D

Reply:
ok great, no problem, maybe something we could do on a live stream? just an idea 🤓

Reply:
sure, we could 👍

Reply:
will we do this in a stream tomorrow then or sometime different

Reply:
We could pair on this on tomorrow's live stream and get people involved, if you think it is a good time? Let me know me know what/when you prefer 🤓